.github/workflows/deploy.yaml

```diff
@@ -35,17 +35,15 @@ jobs:
         uses: anantaramdas/ipfs-pinata-deploy-action@a551f37c17ec3961df7c3ad52e1cd266dfd3d7e7
         with:
           pin-name: Uniswap Default Token List
-          path: 'build/uniswap-default.tokenlist.json'
+          path: "build/uniswap-default.tokenlist.json"
           pinata-api-key: ${{ secrets.PINATA_API_KEY }}
           pinata-secret-api-key: ${{ secrets.PINATA_API_SECRET_KEY }}
 
       - name: Update DNS with new IPFS hash
         env:
           CLOUDFLARE_TOKEN: ${{ secrets.CLOUDFLARE_TOKEN }}
-          RECORD_DOMAIN: 'uniswap.org'
-          RECORD_NAME: '_dnslink.tokens'
           CLOUDFLARE_ZONE_ID: ${{ secrets.CLOUDFLARE_ZONE_ID }}
-        uses: textileio/cloudflare-update-dnslink@0fe7b7a1ffc865db3a4da9773f0f987447ad5848
+          CLOUDFLARE_GATEWAY_ID: ${{ secrets.CLOUDFLARE_GATEWAY_ID }}
```

Comment:
how does the action know which specific subdomain/dnsLink to update?

Reply:
the gateway id corresponds to a specific subdomain https://developers.cloudflare.com/web3/ipfs-gateway/concepts/dnslink/

Reply:
ah gotcha